includes/Checker/Default_Check_Repository.php

```diff
@@ -103,6 +103,23 @@ function ( $slug ) use ( $checks ) {
 			);
 		}
 
-		return $checks;
+		// Return all checks, including experimental if requested.
+		if ( $flags & self::INCLUDE_EXPERIMENTAL ) {
+			return $checks;
+		}
+
+		// Remove experimental checks before returning.
+		return array_filter(
+			array_map(
+				function ( $check ) {
+					if ( $check->get_stability() === Check::STABILITY_EXPERIMENTAL ) {
+						return false;
+					}
+
+					return $check;
+				},
+				$checks
+			)
+		);
```

Comment:
I think `array_map()` here is unnecessary since you're not modifying the result?
```suggestion
		return array_filter(
			$checks,
			static function ( $check ) {
				return $check->get_stability() !== Check::STABILITY_EXPERIMENTAL;
			}
		);
```

Reply:
Thanks @felixarntz I've made this update